src/yetibot/api/google.clj

```diff
@@ -0,0 +1,83 @@
+(ns yetibot.api.google
+  (:require
+   [taoensso.timbre :refer [warn info]]
+   [clojure.string :as s]
+   [clojure.data.json :as json]
+   [clj-http.client :as client]
+   [yetibot.core.config :refer [conf-valid? config-for-ns]]))
+
+(def config (config-for-ns))
+
+(defn configured? []
+  (let [config-keys   [:api-key :custom-search-engine-id]
+        config-values (select-keys config config-keys)]
+    (and (conf-valid? config)
+       (= (count config-values) 2))))
+
+(defonce api-url "https://www.googleapis.com/customsearch/v1?parameters")
+
+;; this has yet to be used somewhere
+(defonce keywords (atom {}))
```

Comment:
I'm curious what the intention is with `keywords` and `accepted-keywords`. Sounds like you wanted to address it in a future PR. That's totally fine.